flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -379,12 +381,11 @@ private String operatorName(String suffix) {
     }
 
     @VisibleForTesting
-    List<Integer> checkAndGetEqualityFieldIds() {
+    List<Integer> checkAndGetEqualityFieldIds(List<String> equalityColumns) {
```

Comment:
We have assigned `this.equalityFieldColumns`, so is there a need to change here?

Reply:
It's for testing. Please check the changes in `[TestFlinkIcebergSinkV2.java](https://github.com/apache/iceberg/pull/8195/files#diff-560cafe3f640ca5cdff37ff1af2fe15d7e87dd644ca2669bf742c2999628bd5d)`